app/src/main/res/layout/activity_maps.xml

```diff
@@ -20,32 +20,35 @@
             android:orientation="horizontal"
             android:paddingTop="@dimen/maps_activity_radius_container_padding"
             android:paddingLeft="@dimen/maps_activity_radius_container_padding"
-            android:paddingRight="@dimen/maps_activity_radius_container_padding">
+            android:paddingRight="@dimen/maps_activity_radius_container_padding"
+            android:paddingBottom="@dimen/maps_activity_radius_container_padding"
+            >
 
             <TextView
-                android:layout_width="wrap_content"
+                android:layout_width="50dp"
                 android:layout_height="wrap_content"
+                android:layout_weight="0"
                 android:text="@string/label_radius_select"
                 android:textSize="@dimen/maps_activity_radius_text_size"
                 android:textStyle="bold" />
 
             <SeekBar
                 android:id="@+id/radius_slider"
                 style="@style/TextAppearance.AppCompat.Large"
-                android:layout_width="match_parent"
+                android:layout_width="fill_parent"
```

Comment:
Why? I thing match_parent is more safe.

Reply:
Match patent take the place of the whole patent, fill parent fill the non declared space